### DISCUSSION TITLE
Allow deletion of own issues

### DISCUSSION THREAD

**Discussion by damil42** [community/community#126059] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

Sometimes I report an issue in a repository (not my own) and have solved it for myself a few seconds or minutes later. Or just want to delete it a later. But I can only close this issue. 

Please allow to delete own issues. At least if there are no answers. Same for questions btw. Why not?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by PavanLuca** [comment]
You might be interested in my discussion 👉 https://github.com/orgs/community/discussions/128602.
